src/utils/LogoMap.ts

```diff
@@ -0,0 +1,14 @@
+import AnywayImage from '../assets/anyway.png';
+import OrYarokLogo from '../assets/Or_Yarok.png';
+
+
+const orYarok = 'אור ירוק'
+const none = null;
+export const anywayLogo = AnywayImage;
+
+const logoMap  : Map<string | null,string>  = new Map();
+logoMap.set(orYarok,OrYarokLogo);
+logoMap.set(none,AnywayImage);
```

Comment:
This is not the way we treat images. also it is not considered "utils"
see `MapIcon.tsx` as example